sinatra-contrib/lib/sinatra/capture.rb

```diff
@@ -106,12 +105,12 @@ def capture(*args, &block)
         dummy      = DUMMIES.fetch(current_engine)
         options    = { :layout => false, :locals => {:args => args, :block => block }}
 
-        buffer.try :clear
+        buffer.public_send(:clear) if buffer.respond_to?(:clear)
```

Comment:
Can I ask why the public send instead of just execute the method if the object responds to `clear`?
```ruby
buffer.clear if buffer.respond_to?(:clear)
```